opendihu_examples/isotonic_contraction/optimize_length_before_prestretch/cuboid_muscle/settings_contraction_with_prestretch.py

```diff
@@ -0,0 +1,686 @@
+scenario_name = "tensile_test"
+dirichlet_bc_mode = "fix_floating"                                                              
+ 
+                                                                         
+starting_length = float(sys.argv[0])
+end_length = float(sys.argv[1])
+prestretch_extension = end_length - starting_length
+if prestretch_extension < 0:
+  print("end_length is smaller than starting_length: Error")
+  quit()
+if prestretch_extension >= 10.0:
+  print("end_length is too large in comparison to starting_length: Error")
+  quit()
+print("scenario_name: {}".format(scenario_name))
+print("starting_length: {}".format(starting_length))
+print("prestretch extension: {}".format(prestretch_extension))
+    
+
+
+nx, ny, nz = 3, 3, 12                     # number of elements
+mx, my, mz = 2*nx+1, 2*ny+1, 2*nz+1 # quadratic basis functions
+
+fb_x, fb_y = 10, 10         # number of fibers
+fb_points = 100             # number of points per fiber
+fiber_direction = [0, 0, 1] # direction of fiber in element
+physical_extent = [3.0, 3.0, starting_length]
+
+def get_fiber_no(fiber_x, fiber_y):
+    return fiber_x + fiber_y*fb_x
+
+meshes = { # create 3D mechanics mesh
+    "3Dmesh_quadratic": { 
+      "inputMeshIsGlobal":          True,                       # boundary conditions are specified in global numberings, whereas the mesh is given in local numberings
+      "nElements":                  [nx, ny, nz],               # number of quadratic elements in x, y and z direction
+      "physicalExtent":             physical_extent,            # physical size of the box
+      "physicalOffset":             [0, 0, 0],                  # offset/translation where the whole mesh begins
+    },
+}
+
+for fiber_x in range(fb_x):
+    for fiber_y in range(fb_y):
+        fiber_no = get_fiber_no(fiber_x, fiber_y)
+        x = nx * fiber_x / (fb_x - 1)
+        y = ny * fiber_y / (fb_y - 1)
+        nodePositions = [[x, y, starting_length * i / (fb_points - 1)] for i in range(fb_points)]
+        meshName = "fiber{}".format(fiber_no)
+        meshes[meshName] = { # create fiber meshes
+            "nElements":            [fb_points - 1],
+            "nodePositions":        nodePositions,
+            "inputMeshIsGlobal":    True,
+            "nRanks":               n_ranks
+        }
+
+# set Dirichlet BC, fix bottom
+elasticity_dirichlet_bc = {}
+contraction_elasticity_dirichlet_bc = {}
+
+
+prestretch_initial_displacements = [[0.0,0.0,prestretch_extension*i/(mx*my*mz-1)] for i in range(mx*my*mz)]
+
+k = 0
+
+# fix z value on the whole x-y-plane
+for j in range(my):
+  for i in range(mx):
+    elasticity_dirichlet_bc[k*mx*my + j*mx + i] = [None,None,0.0,None,None,None]
+    elasticity_dirichlet_bc[(mz-1)*mx*my + j*mx + i] = [None,None,prestretch_extension,None,None,None]
+
+    contraction_elasticity_dirichlet_bc[k*mx*my + j*mx + i] = [None,None,0.0,None,None,None]
```

Comment:
```suggestion
    contraction_dirichlet_bc[k*mx*my + j*mx + i] = [None,None,0.0,None,None,None]
```
Word elasticity doesnt really add any new info